docs/resources/tozny_realm_group_with_access_policy.md

```diff
@@ -0,0 +1,143 @@
+# tozny_realm_group Resource with access_policy
+
+Resource for provisioning a TozID Group with an attached Access Policy to enable multi-party control.
```

Comment:
`with optional configuration such as...`

Reply:
actually since this isn't a standalone resource, in order for it to show up on the registry it needs to be merged in with

https://github.com/tozny/terraform-provider-tozny/blob/master/docs/resources/tozny_realm_group.md

Reply:
https://registry.terraform.io/providers/tozny/tozny/latest/docs/resources/tozny_realm_group

Reply:
https://www.terraform.io/registry/providers/docs

Reply:
Merged documentation with `tozny_realm_group.md`, deleted `tozny_realm_group_with_access_policy.md`